### DISCUSSION TITLE
Welcome to pi-node Discussions!

### DISCUSSION THREAD

**Discussion by KOSASIH** [KOSASIH/pi-supernode#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ramanerayi** [comment]
Selamat sore bro Kosasih,,
Salam kenal, sya Herman Pioneer Indonesia,,
Mhn dibantu utk Spesifikasi Computer Node itu lengkapnya seperti apa yaa, terimakasih atas perhatiannya,,

**Reply by nuhushafiu070-hub** [reply]
What is the merits(advantage) of refuse to release pi Dex and GCV now sir?
